Mobl_ios_Callisto-master/MealTrackingPilot/LogCommonMeal/Views/OccasionPicker.swift

```diff
@@ -597,90 +482,88 @@ open class PickerView: UIView {
             let middleItem = Int(ceil(Float(numberOfItemsByDataSource) / 2.0))
             indexForSelectedItem = middleIndex - (numberOfItemsByDataSource - middleItem)
         }
-
+        
         return indexForSelectedItem
     }
-
+    
     open func selectItem(_ item : Int, animated: Bool) {
```

Comment:
remove the open from all the statements, this will use only when used from library

Reply:
Done